internal/status/svc_status.go

```diff
@@ -60,44 +60,43 @@ func (l *leader) UpdateL4LBStatus(options []UpdateOptions, bulk bool) {
 			if len(svcMetadata.HostNames) > 0 {
 				svcHostname = svcMetadata.HostNames[0]
 			}
+
 			for _, vip := range option.Vip {
-				service.Status = corev1.ServiceStatus{
-					LoadBalancer: corev1.LoadBalancerStatus{
-						Ingress: []corev1.LoadBalancerIngress{{
-							IP:       vip,
-							Hostname: svcHostname,
-						}}}}
+				service.Status.LoadBalancer.Ingress = append(
```

Comment:
@aaha97 This is tested to work for the first operation right? Just had this doubt, otherwise changes look good.

Reply:
Consider the case when we had an IpV4 and IPv6 already in place. Now if lets say request for updating IPV4 to a new Ip comes in. Will this still update the existing IPv4 or just append it at the end ?

Reply:
@DixitAakash @pkoshtavmware have updated it to set list to empty first, thanks

Reply:
Is this tested ?
As in does rest layer send `[existingip, newip]` in `option.Vip` or just `[existingip]` .